browser/src/core/Socket.js

```diff
@@ -1301,6 +1304,10 @@ app.definitions.Socket = L.Class.extend({
 			window.wopiSrc = this._map.options.wopiSrc;
 
 			if (textMsg.startsWith('renamefile:')) {
+				// start loading animation
+				$('#document-name-input').prop('disabled', true);
```

Comment:
could you put that code related to document name input into file:
https://github.com/CollaboraOnline/online/blob/master/browser/src/control/Control.DocumentNameInput.js

(create method which will be called from here)

To keep related things together.